Flutter/assistview/conversation-area.md

```diff
@@ -108,6 +536,89 @@ Append a toolbar to response messages that provides options to perform various a
 {% tabs %}
 {% highlight dart %}
 
+  late List<AssistMessage> _messages;
+
+  void _generativeResponse(String data) async {
+    final String response = await _getAIResponse(data);
+    setState(() {
+      _messages.add(AssistMessage.response(
+        data: response,
+        toolbarItems: [
+          const AssistMessageToolbarItem(
+            content: Icon(Icons.thumb_up_outlined),
+            tooltip: 'Like',
+          ),
+          const AssistMessageToolbarItem(
+            content: Icon(Icons.thumb_down_outlined),
+            tooltip: 'DisLike',
+          ),
+          const AssistMessageToolbarItem(
+            content: Icon(Icons.copy_all),
+            tooltip: 'Copy',
+          ),
+          const AssistMessageToolbarItem(
+            content: Icon(Icons.restart_alt),
+            tooltip: 'Restart',
+          ),
+        ],
+      ));
+    });
+  }
+
+  Future<String> _getAIResponse(String data) async {
+    String response = '';
+    // Connect with your preferred AI to generate a response to the request.
+    return response;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SfAIAssistView(
+        messages: _messages,
+        actionButton: AssistActionButton(
+          onPressed: (String data) {
+            setState(() {
+              _messages.add(AssistMessage.request(data: data));
+            });
+            _generativeResponse(data);
+          },
+        ),
+        onBubbleToolbarItemSelected:
+            (selected, messageIndex, toolbarItem, toolbarItemIndex) {
+          setState(() {
+            _messages[messageIndex].toolbarItems![toolbarItemIndex] =
+                toolbarItem.copyWith(
+              isSelected: selected,
```

Comment:
Wrong details.

like and dislike - selectable 

Others not selectable

Reply:
You can leave this callback empty with todo comment as like we added in AI response genrate method